truffle-config.js

```diff
@@ -165,7 +171,13 @@ module.exports = {
 
     compilers: {
         solc: {
-            version: "0.6.11"
+            version: "0.6.11",
+            settings: {
+                optimizer: {
+                    enabled: true,
+                    runs: 1500
+                }
+            }
```

Comment:
do we need to re-enable optimizer?

Reply:
yep, it was giving issues while fork testing, I don't see an issue with it many smart contracts have this enabled